packages/web-app-files/src/views/spaces/Project.vue

```diff
@@ -339,15 +329,15 @@ export default {
     displayThumbnails() {
       return !this.configuration.options.disablePreviews
     },
-    peopleCountString() {
-      const translated = this.$ngettext(
-        '%{count} member',
-        '%{count} members',
-        this.currentFileOutgoingCollaborators.length
-      )
+    memberCount() {
+      const roles = Object.values(this.space.spaceRoles)
+      return roles.reduce((arr, obj) => arr.concat(obj), []).length
```

Comment:
two proposals:
- 1: you can directly work on `Object.values`, no need to assign it to a variable first
- 2: you can already do the counting in the reducer, like this:
```js
return Object.values(this.space.spaceRoles).reduce((sum, obj) => sum + obj.length, 0)
```

Reply:
Thx, will try! How about including this in the space resource object as `spaceMembers`? I have (at least) another PR which needs exactly the same code. On the other hand, I'm not sure how much we should bloat the space object 🤔

Reply:
IMO fine to include it. `spaceMemberCount` then? `spaceMembers` is a bit misleading, I'd expect an array or object from that.

Reply:
Ah, yes!